packages/bootleg-app-contracts/test/BootlegTraderApp.test.js

```diff
@@ -0,0 +1,132 @@
+const BootlegTraderApp = artifacts.require('BootlegTraderApp');
+const BootlegToken = artifacts.require('BootlegToken');
+
+contract('BootlegTraderApp', accounts => {
+  let tokenContract;
+  let app;
+  const tokenId = web3.utils.toBN(1);
+  const mockURI = 'https://ipfs.infura.io/ipfs/QmSomeHash';
+  const firstOwner = accounts[0];
+  const secondOwner = accounts[1];
+  const thirdOwner = accounts[2];
+  const notAnOwner = accounts[9];
+  const oneEthInWei = web3.utils.toWei('1', 'ether');
+  const twoEthInWei = web3.utils.toWei('2', 'ether');
```

Comment:
I ❤️  these definitions up here. Makes the test cases clear.

Also, fabulous job crafting the describe blocks. Very easy read.

Reply:
Thanks! I do love me some Describe blocks... #rspec4life